src/Common/Core/Impl/UI/BindableBase.cs

```diff
@@ -5,7 +5,7 @@
 using System.ComponentModel;
 using System.Runtime.CompilerServices;
 
-namespace Microsoft.Common.Wpf {
+namespace Microsoft.R.Common.Core.UI {
```

Comment:
I'm not a big fan of using `INotifyPropertyChanged` in non-UI context, but if you want it to be in `Common.Core`, I think we should rename it into `PropertyChangedBase` or something like that and change its namespace.